File: test/fixtures/dart/delete.dart
import 'package:http/http.dart' as http;

void main() async {
  final url = Uri.parse('http://localhost:28139/page');

  final res = await http.delete(url);
  final status = res.statusCode;
  if (status != 200) throw Exception('http.delete error: statusCode= $status');

  print(res.body);
}

File: test/fixtures/rust/delete.rs
extern crate reqwest;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let client = reqwest::blocking::Client::builder()
        .redirect(reqwest::redirect::Policy::none())
        .build()
        .unwrap();
    let res = client.delete("http://localhost:28139/page")
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}
